src/CanCommunity.ts

```diff
@@ -239,9 +239,14 @@ export class CanCommunity {
       return true;
     }
 
-    // TODO check the case that is_any_community_member
-    if (rightHolder.is_any_community_member || rightHolder.is_anyone) {
-      return true;
+    if (rightHolder.is_any_community_member) {
+      const communityMember = await this.query(TableNameEnum.V1_MEMBER, {
+        upper_bound: account,
+        lower_bound: account,
+      });
+      if (communityMember && communityMember.rows && communityMember.rows[0] && communityMember.rows[0].member === account) {
```

Comment:
Have you ever try `communityMember?.rows?.[0]?.member ` for shorter?

Reply:
https://github.com/tc39/proposal-optional-chaining

Reply:
Nice